docker-compose.yml

```
@@ -56,15 +71,139 @@ services:
     driver: bridge
+    ipam:
+      config:
+        - subnet: 172.20.0.0/16
 
 volumes:
-  localstack-data: 
\ No newline at end of file
+  localstack-data:
+  redis-data:
+  grafana-data:
+  prometheus-data: 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)